docs/release-notes/7.17.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[release-notes-header-7.17.0]]
+== 7.17
+
+[discrete]
+[[release-notes-7.17.0]]
+=== 7.17.0
+
+[discrete]
+[[bug-fixes-7.17.0]]
+==== Bug fixes and enhancements
+* Adds detailed telemetry statistics for legacy and regular notifications ({pull}123332[#123332], {pull}122472[#122472]).
+* Fixes a bug that changes the message in the *Activity Log* tab when users re-fetch log data for a date range without data ({pull}123039[#123039]).
+* Updates privilege checks on the *Exceptions* tab ({pull}122902[#122902]).
+* Removes leftover alert notifications after a rule is deleted ({pull}122610[#122610]).
+* Enables cross-space telemetry for cases ({pull}122477[#122477]).
```

Comment:
```suggestion
* Enables cross-space telemetry for Cases ({pull}122477[#122477]).
```
Not sure if Cases should be capitalized as below. Maybe not since this is just referring to the feature in general rather than the Cases table or page, but figured I'd check.